.eslintrc.js

```diff
@@ -209,5 +209,7 @@ module.exports = {
                 ],
             },
         ],
+
+        'prettier/prettier': ['error', { endOfLine: 'auto' }],
```

Comment:
this can also be pulled into another separate PR, I assume its because you're using windows?